syntax = "proto3";

package artist;

service ArtistService {
    rpc GetDescription (GetDescriptionRequest) returns (GetDescriptionResponse);
    rpc CreateArtist (CreateArtistRequest) returns (CreateArtistResponse);
}

message GetDescriptionRequest {
    int32 id = 1;
}

message GetDescriptionResponse {
    string description = 1;
}

message CreateArtistRequest {
    string name = 1;
    string description = 2;
}

message CreateArtistResponse {
    int32 id = 1; // ID созданного исполнителя
}